pkg/ebpf/offsetguess.go

```diff
@@ -80,9 +76,20 @@ var whatString = map[C.__u64]string{
 	guessSport:     "source port",
 	guessDport:     "destination port",
 	guessNetns:     "network namespace",
+	guessRTT:       "Round Trip Time",
 	guessDaddrIPv6: "destination address IPv6",
 }
 
+const (
+	tcpInfoKProbeNotCalled C.__u64 = 0
```

Comment:
For consistency shouldn't both (or neither) be `C.__u64` - on assignment to the `tcpKprobeCalledString` they might be transparently casted.

Reply:
So I kept it this way for consistency with the rest of the file.